src/tests/unit/tests/DetailsView/components/tab-stops-failed-instance-section.test.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { VisualizationScanResultData } from 'common/types/store-data/visualization-scan-result-data';
+import {
+    TabStopsFailedInstanceSection,
+    TabStopsFailedInstanceSectionDeps,
+    TabStopsFailedInstanceSectionProps,
+} from 'DetailsView/components/tab-stops-failed-instance-section';
+import { TabStopsFailedCounter } from 'DetailsView/tab-stops-failed-counter';
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { IMock, Mock } from 'typemoq';
+
+describe('TabStopsFailedInstanceSection', () => {
+    let tabStopsFailedCounterMock: IMock<TabStopsFailedCounter>;
+
+    const visualizationScanResultDataStub = {
+        tabStops: { requirements: {} },
+    } as VisualizationScanResultData;
+
+    let props = {} as TabStopsFailedInstanceSectionProps;
```

Comment:
You can actually not set props or deps here, if you like, since you don't use them as empty objects anyways.